reflex-dhtmlx/src/Reflex/Dom/DHTMLX/Date.hs

```diff
@@ -119,4 +123,4 @@ dhtmlxDatePicker (DatePickerConfig iv sv b p wstart attrs visibleOnLoad) = do
       performEvent_ $ dateWidgetHide cal <$ ups
       return ups
     let parser = parseTimeM True defaultTimeLocale fmt . T.unpack
-    fmap DatePicker $ holdDyn iv $ parser <$> leftmost [_textInput_input ti, ups]
+    fmap DatePicker $ holdDyn iv $ leftmost [parser <$> leftmost [_textInput_input ti, ups], sv]
```

Comment:
I think it's better to collapse this into one `leftmost`.

Reply:
How would you propose? the alternative I can think of is this
```haskell
leftmost [parser <$> _textInput_input ti, parser <$> ups, sv]
```
Do you prefer that?

Reply:
Yeah, fewer merges.